app/src/main/java/org/traccar/client/MainActivity.java

```diff
@@ -307,4 +289,38 @@ public void onRequestPermissionsResult(int requestCode, String[] permissions, in
         }
     }
 
+    private boolean validateServerURL(String userUrl) {
+        if (userUrl == null || userUrl.trim().length() == 0) {
```

Comment:
If someone explicitly enters "nothing" it makes more sense to use the default value than to just reject the action, imho.